secure-staff-services.md

```diff
@@ -0,0 +1,65 @@
+
+This approach allows us to authenticate external requests before they reach our applications.
+
+A proposed extension to this would be this [Cognito authentication proposal](https://docs.google.com/document/d/1bEyL_Hn7DRs7XNyhvjvDK0PGXRzxgFzCbMcnaVyXbA0/edit#heading=h.nchxv8vlw2w5) that provides further custom authentication with Google as an IdP but allows potential access management based on Google groups, and enforces MFA requirements. 
+
+This will remove the requirement to perform authentication within application code and provide a standard way of authenticating all our applications that can easily be [encoded in a GuCDK pattern](https://docs.google.com/document/d/1SfvjNRIzv1bNYRho7s5i_YZv6YyWadp4j9rNOqvpnJg/edit#heading=h.812bucjdpfq7).
+
+### Web Application Firewall (WAF)
+
+To provide a standard way to recognise and prevent common attacks than can come via web requests we will use AWS WAF [Web Access Control Lists (ACLs)](https://docs.aws.amazon.com/waf/latest/developerguide/web-acl.html). These can be integrated with various AWS solutions including AWS ALBs. See the following [Architecture Decision Record](https://github.com/guardian/waf/blob/main/adr/waf-management.md) for further context.
+
+This solution allows centrally managing Web ACL rules so that they are easily understood, contain an audit trail via source control and can easily be reviewed by InfoSec. In addition this solution can provide a [stream of request events](https://github.com/guardian/waf/blob/main/lib/primary-waf.ts#L62) that can be centrally audited.
+
+### Authentication at the application
+
+In order to ensure that requests that reach services (EC2 instances or Lambdas) inside our networks we should re-authenticate requests either by validating any tokens generated at the ALB or entirely re-authenticating the user.
+
+Ensuring that we re-authenticate internal requests prevents lateral movement of attacks inside our network should external access be compromised.
+
+This can be done by verifying the token passed from the ALB during the authentication step (no current example) or by re-authenticating the user in the application using an existing auth library such as [guardian/play-googleauth](https://github.com/guardian/play-googleauth) (Scala), or [guardian/pan-domain-authentication](https://github.com/guardian/pan-domain-authentication/#to-verify-login-in-nodejs) (Node). 
+
+In the future we hope to develop a common pattern for verifying tokens from the ALB to prevent the need to re-authenticate at the application.
+
+### Authorisation / Access Management
+
```

Comment:
Maybe expand here on how to categorise a service and the appropriate level of security.